scrapers.js

```diff
@@ -1424,10 +1424,13 @@ let scrapers = [
     county: 'Glenn County',
     state: 'CA',
     country: 'USA',
-    url: 'https://www.countyofglenn.net/dept/health-human-services/public-health/welcome',
+    url: 'https://www.countyofglenn.net/dept/health-human-services/public-health/covid-19',
     scraper: async function() {
       let $ = await fetch.page(this.url);
```

Comment:
```suggestion
      if (datetime.scrapeDateIsBefore('2020-3-16')) {
        this.url = 'https://www.countyofglenn.net/dept/health-human-services/public-health/welcome';
      }
      else {
        this.url = 'https://www.countyofglenn.net/dept/health-human-services/public-health/covid-19';
      }

      let $ = await fetch.page(this.url);
```

For backwards compatibility